src/utils/getWADORSImageId.js

```diff
@@ -6,7 +6,7 @@ function getWADORSImageUrl(instance, frame) {
   }
 
   // We need to sum 1 because WADO-RS frame number is 1-based
-  frame = (frame || 0) + 1;
+  frame = frame ? parseInt(frame) + 1 : 1;
```

Comment:
Always have to double check my understanding of falsey.

```
// 1
undefined, null, empty string, 0, "0", 

// 2
1, "1", and "01"
```

Reply:
@dannyrb You're right here - we should probably use an `if (frame === undefined || frame === null)` instead of the ternary here just to improve readability and make sure the execution path is predictable.

Reply:
Actually, the falsy check makes more sense. Because 0 should yield 1 and so should ''. I'm going to leave this for the moment if you're okay with it. In the future, I'll probably avoid the ternary. Always confuses me.